### PR TITLE
try-o-rama stress tests

### PR DESCRIPTION
**Pull request by maackle**
## PR summary

Basic implementation of some stress tests. There are two tests which do the same thing, one in series, one in parallel. Both are parameterized by two variables, the number of conductors (N) and the number of instances per conductor (M)

- spin up N conductors with M instances each
- have each instance commit a unique entry
- have one instance commit another entry and link all other entries onto it
- have each instance read links and verify that they see all M * N entries

This is similar to the existing cluster tests, but leveraging try-o-rama.

## followups

Once we know what kinds of stress tests we'll want to run for various hApps, it will be time to build a bit of a framework for making it easy to stress test a new hApp.

## changelog

Please check one of the following, relating to the [CHANGELOG-UNRELEASED.md](https://github.com/holochain/holochain-rust/blob/develop/CHANGELOG-UNRELEASED.md)

- [ ] this is a code change that effects some consumer (e.g. zome developers) of holochain core so it is added to the CHANGELOG-UNRELEASED.md (linked above), with the format `- summary of change [PR#1234](https://github.com/holochain/holochain-rust/pull/1234)`
- [x] this is not a code change, or doesn't effect anyone outside holochain core development

### CODE REVIEW

**Comment by maackle** [comment]
These might still be flaky. It might be helpful to be more clear on what we want in stress tests before merging this. Once we merge this we should be committed to keeping it green, for whatever tests are in here, we should choose wisely.

File: app_spec/cluster_test/default.nix [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
+  script-cluster = pkgs.writeShellScriptBin name-cluster
   ''
   hc-cli-install
   hc-conductor-install
    ( cd hc_cluster_test && npm install && ./node_modules/.bin/tsc)
    ( cd app_spec && mkdir -p dist && hc package --output dist/app_spec.dna.json --strip-meta )
    ( EMULATION_HOLOCHAIN_BIN_PATH=./.cargo/bin/holochain node ./app_spec/cluster_test/index.js 2)
   '';
+
+  name-stress = "hc-stress-test-sim1h";
```
Comment (lucksus) [inline app_spec/cluster_test/default.nix]: Should this rather be in `stress-test/default.nix`?

Comment (maackle) [reply app_spec/cluster_test/default.nix]: Yeah good call.

**Comment by maackle** [comment]
Stress tests are now parameterized from the command line, so you can do for instance:

    hc-stress-test-sim2h 50 2

which will run stress tests with 50 conductors, 2 instances per conductor

**Pull request merged**